components/drafts/draft_actions/action.scss

```diff
@@ -0,0 +1,52 @@
+.DraftAction {
+    &__button {
+        display: inline-flex;
+        width: 28px;
+        height: 28px;
+        align-items: center;
```

Comment:
```
       align-items: center;
        justify-content: center;
```
FYI: We could use 'place-items: center'

Reply:
`place-items: center` doesn't centre the items when I test it locally

Reply:
sorry, it's `place-content` I am always confused with this.